lib/active_record/tasks/sqlserver_database_tasks.rb

```diff
@@ -14,7 +14,7 @@ class SQLServerDatabaseTasks
                to: ActiveRecord::Base
 
       def initialize(configuration)
-        @configuration = configuration
+        @configuration = ActiveSupport::HashWithIndifferentAccess.new(configuration)
```

Comment:
Why did you have to change it to a `HashWithIndifferentAccess`? Are you trying to mimic the new `using_database_configurations?` we should implement?

Reply:
After the 6.1 upgrade, none of the string keys were resolving. All of the keys were symbols. 

Changing it to a hash with indifferent access let me fix it with the least amount of disruptions.